01-syntax/02-functions-variables-scope-and-memory/bean-counting/countChar.js

```diff
@@ -4,7 +4,18 @@
  */
 /* global countChar */
 
-// →Your code here.
+function countChar(text, character) {
+  var charCount;
+  var numberOfCharacters = 0;
+
+  for (charCount = 0; charCount < text.length; charCount++) {
+    if (text.charAt(charCount) === character) {
+      numberOfCharacters += 1;
+    }
+  }
+
+  return numberOfCharacters;
+}
 
 console.log(countChar('kakkerlak', 'k'));
```

Comment:
The same here, try to think about edge cases, on how function could be invoked by other developer